packages/devtools/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  devtools_server: ^0.1.15
+  devtools_server: ^0.2.2
```

Comment:
once I release devtools_server 0.2.3 (hopefully today), will I need to bump this to 0.2.3?

Reply:
I think that's less of a priority as Flutter doesn't depend on it, but it probably wouldn't hurt.